src/components/ListBox/ListBox.js

```diff
@@ -6,6 +6,10 @@ import ListBoxMenu from './ListBoxMenu';
 import { ListBoxType } from './ListBoxPropTypes';
 import childrenOf from '../../prop-types/childrenOf';
 
+const handleOnKeyDown = event => {
+  event.stopPropagation();
```

Comment:
Should we explicitly check for ESC key…?

Reply:
Good catch! Will add that in 🙂